blockchain/integrity/integrity_test.go

```diff
@@ -244,13 +244,19 @@ func addTestingConstantinopleBlocks(bc blockchain.Blockchain, dao blockdao.Block
 }
 
 func addTestingTsfBlocks(cfg config.Config, bc blockchain.Blockchain, dao blockdao.BlockDAO, ap actpool.ActPool) error {
+	ctx := context.Background()
+	addOneTx := func(tx action.SealedEnvelope, err error) error {
```

Comment:
It is strange to pass `error` as input.
I would prefer to create `addOneTsf` and `addOneExec` separately. The pseudocode code looks like 
```
addOneTsf := func(...) error {
	tsf, err := action.SignedTransfer(...)
	if err != nil {
		return err
	}
	return ap.Add(context.Background(), tsf);
}

```